STACK-UP-3/champs-backend
# 171648169 Feature of signing in a user

#### Description
> This PR adds the sign-in feature to the application
#### Type of change

Please select the relevant option


#### How Has This Been Tested?


#### How to Test
> Use postman to test the sign-in feature,
Give instructions on how reviewers can verify your work

#### Any background context you want to add (Optional)
> N/A
Screenshots or GIFs
> Mockup
https://www.figma.com/file/2KDdSdzTye8EG7l0Wxl36D/Barefoot-dev?node-id=1%3A3

>Implementation plan
https://docs.google.com/document/d/134cKdaAu0KEDQYztmDO0ljIuTO16WvA_eeDhhUhMLPE/edit?usp=sharing
#### STORY ID

[[Finishes #171648169]
